tests/integration/test_dbfs_store/test_dbfs_store.py

```diff
@@ -50,65 +52,59 @@ class TestDBFSStore:
     config_file_path = here / "test-dbfs-store.yml"
     with config_file_path.open() as fp:
         config = yaml.safe_load(fp)
-    parquets_dir = "/parquets"
-    csv_dir = "/csv"
-    test_file_path = str(here / "test.txt")
-    json_path = str(here / "test_data.json")
-    parquet_path = str(here / "test_data.parquet")
-    additional_parquet_path = str(here / "additional_data.parquet")
-    csv_path = str(here / "test_data.csv")
-    additional_csv_path = str(here / "additional_data.csv")
```

Comment:
Moving this stuff to `setup_class` doesn't sit well with PyCharm. Why not leave it where it is?

And of course, I'm referring to all occurrences in this PR.

Reply:
I removed unnecessary parts of the code, leaving only what's essential. I moved the essential part to setup_class in order to serve multiple purposes:

1) It runs once and can be utilized across the entire class.
2) I'm creating a base class that every test class for each datastore will utilize.
Establishing standards for each subclass requires setting up these class attributes.

This is the first step in order to do so.